x/auction/genesis.go

```diff
@@ -10,7 +10,9 @@ import (
 // DefaultGenesis creates a default genesis state
 func DefaultGenesis() *GenesisState {
 	return &GenesisState{
-		RewardsParams:   RewardsParams{BidDuration: 14 * 24 * 3600}, // 14 days
+		// check bid duration value for v6.6 upgrade
+		// RewardsParams:   RewardsParams{BidDuration: 14 * 24 * 3600}, // 14 days
+		RewardsParams:   RewardsParams{BidDuration: 4 * 3600}, //  4 hours for bid duration this is only for canon-4 network
```

Comment:
this is only for `canon-4` network to test auctions , we need more auction records to test the UI and indexer is working or not